base/variables.tf

```
@@ -1,19 +1,3 @@
-variable "region" {
-  description = "AWS region"
-  type        = string
-  default     = "ap-southeast-1"
-}
-
-variable "environment" {
-  description = "The application environment, used to tag the resources, e.g. `acme-web-staging`"
-  type        = string
-}
-
-variable "owner" {
-  description = "The owner of the infrastructure, used to tag the resources, e.g. `acme-web`"
-  type        = string
-}
-
 variable "aws_access_key" {
   description = "AWS Access Key"
   type        = string
```

sensitive vars must be set as sensitive to prevent Terraform from showing its value in the plan or apply output
```suggestion
variable "aws_access_key" {
  description = "AWS Access Key"
  type        = string
  sensitive = true
```

---

I added that option in 5c10aa6 ✅